components/edit-collective/sections/BankTransfer.js

```diff
@@ -82,8 +88,21 @@ const BankTransfer = props => {
 
   const existingManualPaymentMethod = !!get(data.host, 'settings.paymentMethods.manual');
   const showEditManualPaymentMethod = !showForm && data.host;
-  const bankAccount = data.host.payoutMethods.find(pm => pm.data.isManualBankTransfer);
-  const useStructuredForm = !existingManualPaymentMethod || (existingManualPaymentMethod && bankAccount) ? true : false;
+  const existingPayoutMethod = data.host.payoutMethods.find(pm => pm.data.isManualBankTransfer);
+  const useStructuredForm =
+    !existingManualPaymentMethod || (existingManualPaymentMethod && existingPayoutMethod) ? true : false;
+  const instructions =
+    get(data.host, 'settings.paymentMethods.manual.instructions') || BANK_TRANSFER_DEFAULT_INSTRUCTIONS;
+  const fixedCurrency =
+    // Fix currency if the existing payout method already matches the collective currency
+    (!existingPayoutMethod || existingPayoutMethod.data.currency === data.host.currency) &&
+    // or if it was already defined by Stripe
```

Comment:
The comment say "or" while the code say `&&`. It's difficult to follow what is being done here.

Reply:
For clarity, we should avoid long chains like that for assignment.

Reply:
Indeed! It lacks clarity and the logic can be simplified.